v2/cf-nodejs-logging-support/src/lib/config/config-core.json

```diff
@@ -2,13 +2,33 @@
     "fields": [
         {
             "name": "logger",
-            "mandatory": true,
             "source": {
                 "type": "static",
                 "value": "nodejs-logger"
             },
             "output": [
-                "msg-log"
+                "msg-log",
+                "req-log"
+            ]
+        },
+        {
+            "name": "written_at",
+            "source": {
+                "type": "meta"
+            },
+            "output": [
+                "msg-log",
+                "req-log"
+            ]
+        },
+        {
+            "name": "written_ts",
+            "source": {
+                "type": "meta"
```

Comment:
The source does not contain any reference to the meta value that is used. Is it working via the `name` value of the field config? If yes, this would make it difficult to reuse the meta functionality -> Imaging a user trying to rename "written_ts" to, e.g., "written_timestamp" via custom configuration. It would not be possible to use the meta feature in this case without code changes.